ipynb/observationTable.md

```diff
@@ -0,0 +1,164 @@
+
```

Comment:
Jose/Luke told me that we should store a markdown copy of the notebook. The single other example in this dir comes as a *.py file. Thus, no notebook at all?!